fluentd/lib/generate_throttle_configs/lib/generate_throttle_configs.rb

```diff
@@ -168,13 +168,26 @@ def create_default_docker(input_conf_file, excluded, log, options={})
   @label @INGRESS
   path "#{cont_logs_path}"
   pos_file "#{cont_pos_file}"
-  time_format #{use_crio == 'true' ? '%Y-%m-%dT%H:%M:%S.%N%:z' : '%Y-%m-%dT%H:%M:%S.%N%Z'}
+  time_format '%Y-%m-%dT%H:%M:%S.%N%Z'
```

Comment:
I don't understand this.  Don't we need this in 3.11 for crio support?

Reply:
Sorry, I missed the time_format change...  Pushed the fix.